persistence/src/main/java/org/mskcc/cmo/metadb/persistence/neo4j/MetadbPatientRepository.java

```diff
@@ -28,4 +28,9 @@ MetadbPatient findPatientByCmoPatientId(
             + "MATCH (s)<-[:HAS_SAMPLE]-(p: Patient) "
             + "RETURN p.metaDbPatientId")
     UUID findPatientIdBySample(@Param("metaDbSampleId") UUID metaDbSampleId);
+
+    @Query("MATCH (s: Sample)<-[:IS_ALIAS]-(sa: SampleAlias{value: $value, namespace: $namespace}) "
+            + "MATCH (s)<-[:HAS_SAMPLE]-(p: Patient) "
+            + "RETURN p")
+    MetadbPatient findPatientBySample(@Param("namespace") String namespace, @Param("value") String value);
```

Comment:
`findPatientBySampleAlias`

Reply:
decide on what naming we should use.. **SampleAlias or **NamespaceValue

Whatever is decided it should also be applied to method below in sample service to match for consistency